packages/shared/profiles/sagas.ts

```diff
@@ -183,52 +197,91 @@ export function* handleFetchProfile(action: ProfileRequestAction): any {
   }
 }
 
-function* getRemoteProfile(userId: string, version?: number) {
-  try {
-    const profiles: { avatars: Avatar[] } = yield call(profileServerRequest, userId, version)
+export function* handleFetchProfiles(action: ProfilesRequestAction): any {
+  const { userIds, version } = action.payload
 
-    let avatar = profiles.avatars[0]
+  const identity: ExplorerIdentity | undefined = yield select(getCurrentIdentity)
 
-    if (avatar) {
-      avatar = ensureAvatarCompatibilityFormat(avatar)
-      if (!validateAvatar(avatar)) {
-        defaultLogger.warn(`Remote avatar for user is invalid.`, userId, avatar, validateAvatar.errors)
-        return null
-      }
+  if (!identity) throw new Error("Can't fetch profile if there is no ExplorerIdentity")
```

Comment:
Why?

Reply:
We just kept the same check that was being performed in the original method to prevent any errors
https://github.com/decentraland/kernel/blob/79edd230d0c2b5cc43b7a6d7a1bcc9e4050b7d3a/packages/shared/profiles/sagas.ts#L147